packages/postgres-database/src/simcore_postgres_database/models/_common.py

```diff
@@ -0,0 +1,22 @@
+import sqlalchemy as sa
+
+
+def column_created() -> sa.Column:
+    return sa.Column(
+        "created",
+        sa.DateTime(),
+        nullable=False,
+        server_default=sa.sql.func.now(),
+        doc="Timestamp auto-generated upon creation",
+    )
+
+
+def column_modified():
```

Comment:
return type

Reply:
THOUGHT: I wonder what would be a good compromise with annotations. In "Robust Python by Viafore" he makes the case that one should only annotate enough so that static-checkers provide the right feedback. According to that, this would be one case in which the annotation would not bring much ...